src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -683,7 +683,7 @@
     it "redirects to back if a referer is there" do
       request.env["HTTP_REFERER"] = root_url # Needed for the redirect_to :back
```

Comment:
Well, yes and no. Yes, because the controller is using it to redirect. And no because in the controller we have `redirect_back(fallback_location: root_path)`, so if the `request.env["HTTP_REFERER"]` is not set, it also redirects to the `root_path` too because of the `fallback_location`.

Taking that into account it may make sense to set `request.env["HTTP_REFERER"]` to something different to root_path to ensure that we are redirecting back really. Removing it is another alternative.